doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/configuring-autoscaling.rst

```diff
@@ -2,55 +2,44 @@
 
 .. _deployment-guide-autoscaler-under-construction:
 
-Autoscaling with Ray
---------------------
+Configuring the Ray Autoscaler
+==============================
 
-Ray is designed to support highly elastic workloads which are most efficient on
-an autoscaling cluster. At a high level, the autoscaler attempts to
-launch/terminate nodes in order to ensure that workloads have sufficient
-resources to run, while minimizing the idle resources.
+This page provides an overview of the parameters for the Ray autoscaler. A Ray cluster by default runs an autoscaler that modifies
+the number of nodes based on the resources requested by the task, actor, or placement group. If there is insufficient resource to serve
```

Comment:
`runs an autoscaler that modifies the number of nodes` -> `runs an autoscaler that launches or removes nodes`.

May be worth advertising that we can be kinda smart about which nodes we add or remove.

Reply:
is this related to maximizing cluster utilization or something different?

Reply:
i'm thinking more about safety in the case of downscaling.

Reply:
sure, what would you like to mention here for the current downscaling behavior?